src/components/settings/DataManagement/index.tsx

```diff
@@ -1,47 +1,75 @@
-import InfoIcon from '@/public/images/notifications/info.svg'
-
 import Track from '@/components/common/Track'
 import { SETTINGS_EVENTS } from '@/services/analytics'
-import { Paper, Grid, Typography, Tooltip, SvgIcon, Button } from '@mui/material'
+import { Paper, Grid, Typography, Button } from '@mui/material'
 import { useState } from 'react'
 import ImportAllDialog from '../ImportAllDialog'
+import { getPersistedState } from '@/store'
+import { SAFE_EXPORT_VERSION } from '../ImportAllDialog/useGlobalImportFileParser'
+import { pendingTxsSlice } from '@/store/pendingTxsSlice'
+import { cookiesSlice } from '@/store/cookiesSlice'
+
+const handleExport = () => {
```

Comment:
In our address book export we have a small dialog which gives a summary of what will be exported and an export button in the dialog.

For consistency we should do it the same way here.